### PR TITLE
feat: WT-2151 Add connect to RemoteConfiguration type

### PR DESCRIPTION
**Pull request by sharonsheah**
### Hi👋, please prefix this PR's title with:

- [ ] `breaking-change:` if you have introduced modification that necessitates immediate adjustments by this SDK's users to their applications, clients, or integrations to avert disruptions to existing features or functionalities.
- [ ] `feat:`, `fix:`, `refactor:`, `docs:`, or `chore:`.

# Summary

# Detail and impact of the change

## Security

# Anything else worth calling out?

### CODE REVIEW

**Review by ZacharyCouchman — APPROVED** [review]

File: packages/checkout/sdk/src/config/remoteConfigFetcher.ts [hunk 1]
```diff
@@ -77,6 +77,9 @@ export class RemoteConfigFetcher {
     // Ensure that the configuration is valid
     this.configCache = this.parseResponse<RemoteConfiguration>(response);
 
+    // eslint-disable-next-line no-console
+    if (this.configCache && this.configCache.connect.walletConnect) console.warn('WalletConnect is enabled');
```
Comment (dreamoftrees) [inline packages/checkout/sdk/src/config/remoteConfigFetcher.ts:81]: Do you think we need this warning here? I was imagining the warning would be useful if they have configured the widgets options with walletConnect but then our remote feature flag is disabled.. we can then warn them `WalletConnect is currently unavailable.` or something along those lines.

**Review by dreamoftrees — COMMENTED** [review]
Nice! just a comment around the warning here.

File: packages/checkout/sdk/src/config/remoteConfigFetcher.ts (hunk 1 above)
Comment (sharonsheah) [reply packages/checkout/sdk/src/config/remoteConfigFetcher.ts:81]: Ohh right - I misunderstood this part. will add this to the other ticket

**Review by ZacharyCouchman — APPROVED** [review]

**Review by dreamoftrees — APPROVED** [review]
👍

**Pull request merged**